applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/app.overlay

```diff
@@ -51,4 +51,12 @@
 			label = "Green LED 3";
 		};
 	};
+
+	hid_dev_0: hid_dev_0 {
+		compatible = "zephyr,hid-device";
+		interface-name = "HID0";
+		protocol-code = "none";
```

Comment:
Maybe we could assign it `mouse` boot protocol code? (for consistency with `nrf5280dk_nrf52840`)